### PR TITLE
Mgmt 2104 run filelock commands from skipper

### PR DESCRIPTION
**Pull request by RazRegev**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -118,12 +118,12 @@ start_minikube:
...
 
 delete_minikube:
-	python3 scripts/indexer.py --action del --namespace all $(OC_FLAG)
+	skipper run python3 scripts/indexer.py --action del --namespace all $(OC_FLAG)
 	minikube delete --all
 	skipper run discovery-infra/virsh_cleanup.py -m
 
 delete_minikube_profile:
-	python3 scripts/indexer.py --action del --namespace $(NAMESPACE) $(OC_FLAG)
+	skipper python3 scripts/indexer.py --action del --namespace $(NAMESPACE) $(OC_FLAG)
```
Comment (tsorya) [inline Makefile:126]: sure it works? maybe 
skipper run 'python3 scripts/indexer.py --action del --namespace $(NAMESPACE) $(OC_FLAG)'

Comment (tsorya) [reply Makefile:126]: u miss run here

File: Makefile [hunk 2]
```diff
@@ -201,10 +201,10 @@ _deploy_nodes:
 	discovery-infra/start_discovery.py -i $(ISO) -n $(NUM_MASTERS) -p $(STORAGE_POOL_PATH) -k '$(SSH_PUB_KEY)' -md $(MASTER_DISK) -wd $(WORKER_DISK) -mm $(MASTER_MEMORY) -wm $(WORKER_MEMORY) -nw $(NUM_WORKERS) -ps '$(PULL_SECRET)' -bd $(BASE_DOMAIN) -cN $(CLUSTER_NAME) -vN $(NETWORK_CIDR) -nM $(NETWORK_MTU) -ov $(OPENSHIFT_VERSION) -iU $(REMOTE_SERVICE_URL) -id $(CLUSTER_ID) -mD $(BASE_DNS_DOMAINS) -ns $(NAMESPACE) -pX $(HTTP_PROXY_URL) -sX $(HTTPS_PROXY_URL) -nX $(NO_PROXY_VALUES) --service-name $(SERVICE_NAME) --vip-dhcp-allocation $(VIP_DHCP_ALLOCATION) --profile $(PROFILE) --ns-index $(NAMESPACE_INDEX) $(OC_PARAMS) $(KEEP_ISO_FLAG) $(ADDITIONAL_PARAMS)
 
 deploy_nodes_with_install:
-	skipper make $(SKIPPER_PARAMS) _deploy_nodes NAMESPACE_INDEX=$(shell bash scripts/utils.sh get_namespace_index $(NAMESPACE) $(OC_FLAG)) NAMESPACE=$(NAMESPACE) ADDITIONAL_PARAMS=-in $(SKIPPER_PARAMS)
+	skipper make $(SKIPPER_PARAMS) _deploy_nodes NAMESPACE_INDEX=$(shell bash skipper run scripts/utils.sh get_namespace_index $(NAMESPACE) $(OC_FLAG)) NAMESPACE=$(NAMESPACE) ADDITIONAL_PARAMS=-in $(SKIPPER_PARAMS)
```
Comment (tsorya) [inline Makefile:204]: why we can't move 
NAMESPACE_INDEX=$(shell bash skipper run scripts/utils.sh get_namespace_index $(NAMESPACE) $(OC_FLAG))
to discovery-infra/start_discovery.py? just mount all u need there

**Comment by RazRegev** [comment]
/hold 
depends on https://github.com/Stratoscale/skipper/pull/135/commits/3ad73c9d515a047c0aade86d38ae1afaca9f437b

**Comment by mhrivnak** [comment]
I tested this PR and it worked.

**Comment by RazRegev** [comment]
/unhold 
updated skipper

**Pull request merged**